apple/RNCWebView.m

```diff
@@ -1036,9 +1036,11 @@ - (void)                    webView:(WKWebView *)webView
       }
 
       NSString *disposition = nil;
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
small detail, but it would be nice to add a comment to clarify this version check as is done elsewhere in `react-native-webview`... it may not always be obvious to folks what `130000` means

e.g.
https://github.com/react-native-webview/react-native-webview/blob/30685edda045f7ba5d1400b415278e1109436dcd/apple/RNCWebViewManager.m#L18

https://github.com/react-native-webview/react-native-webview/blob/30685edda045f7ba5d1400b415278e1109436dcd/apple/RNCWebViewManager.m#L81

Reply:
You're right, this is also consistent with the rest. I've added one commit to include the comment.